test/runtests.jl

```diff
@@ -5,14 +5,14 @@ aif = detect_ambiguities(ImageFiltering, Kernel, KernelFactors, Base)
 asa = detect_ambiguities(StaticArrays, Base)
 @test isempty(setdiff(aif, asa))
 
+include("mapwindow.jl")
```

Comment:
Is this deliberate or simply a temporary convenience?

Reply:
Just a convenience, I can restore previous order if this is finished.